Makefile

```diff
@@ -66,6 +67,11 @@ test: lint pythoncheck
 lint: pythoncheck
 	flake8
 
+typecheck: pythoncheck
```

Comment:
We should add this to the `make test` command right?

It should probably also be specified in the `.PHONY`

Reply:
Yes for `.PHONY`

Not yet for make test, because it breaks too much for now

Reply:
I actually think, someone should focus on mypy for 2 days, fix everything ad add it to `make test` and force other contributors to comply afterward.

Reply:
How about we land this in master and then open another issue for converting the rest of the code base?

Reply:
Agree

Reply:
👍 Let's get the PHONY updated and an issue, then I'll 🍏 :)